### ISSUE TITLE
name.bas has a missing character

### ISSUE THREAD

**Issue by Rev-3ng** [GReaperEx/bcg#7]
missing ')' in line 20 and also missing in the original BASIC Computer Games book.

**Comment by GReaperEx** [comment]
Interesting... If it is the same in the book, should I change it?...

**Comment by Rev-3ng** [comment]
Whether you decide to edit it or not is obviously your call. If you leave it as is, perhaps add a note somehow.

I've got an original 1975 version of 101 BASIC Computer Games by Digital Corporation (Blue and White only cover). This program is not included in that version.
It is included in the later BASIC Computer Games - Microcomputer Edition which appears to be 1978 edition. scans available on-line with the error showing.
It is also in the BASIC Computer Games - TRS-80 Edition (also ~1978 edition) but has been renumbered and fixed in that version. I have included a scan of that cover and listing for your info.

[BASIC Computer Games - TRS-80 Edition.PDF](https://github.com/GReaperEx/bcg/files/9276473/BASIC.Computer.Games.-.TRS-80.Edition.PDF)

[NAME.BAS.PDF](https://github.com/GReaperEx/bcg/files/9276469/NAME.BAS.PDF)

sorry about the scan not showing the line numbers clearly. it's awkward trying to hold the book flat to scan without damaging it.

regards,
Mark.
